dev/com.ibm.ws.jsfContainer_fat_2.3/test-applications/JSF22CDIFacesFlows/resources/WEB-INF/beans.xml

```diff
@@ -13,5 +13,5 @@
 <beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
-       bean-discovery-mode="all">
+       version="1.0" bean-discovery-mode="all">
```

Comment:
```suggestion
       version="1.1" bean-discovery-mode="all">
```

Reply:
Thanks. For future instances where there is no version in the beans.xml, should I set it to 1.1 or does it depend?

Reply:
set it to match the version in the schema... in this case `http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd`
note that CDI 1.0 does not support the version attribute (or bean-discovery-mode)

Reply:
I don't know how I missed that. Thank you